config/local-setup/controller/managed_zone.yaml

```diff
@@ -6,3 +6,8 @@ spec:
   id: DUMMY_ID
   domainName: DUMMY_DOMAIN_NAME
   description: "Dev Managed Zone"
+  provider:
+    name: mgc-dns-provider
```

Comment:
yeah def find this a bit odd, for the provider not to specify which provider it is but instead to specify that in the zone